include/bout/array.hxx

```diff
@@ -375,17 +375,9 @@ private:
    * @param[in] cleanup   If set to true, deletes all dataBlock and clears the store
    */
   static storeType& store(bool cleanup = false) {
-#ifdef _OPENMP
     static arenaType arena(omp_get_max_threads());
-#else
-    static arenaType arena(1);
-#endif
```

Comment:
we can certainly change no OpenMP -> `0` - but than this needs to be reverted.

It is also very convinient to calculate the total number of threads, just multiply `omp_get_max_threads() * NXPE * NYPE`.

What would be the use case for using `0`?

Reply:
Is there another way that we can distinguish between no OpenMP and OpenMP with one thread? If there's another flag to say whether we are using OpenMP, perhaps renaming this from `omp_get_max_threads` to just `max_threads()`, since the threads may not be OpenMP threads?